apps/web/src/modules/auction/components/AuctionPaused.tsx

```diff
@@ -0,0 +1,88 @@
+import { Box, Stack, atoms } from '@zoralabs/zord'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import { useMemo } from 'react'
+import useSWR from 'swr'
+import { useContract, useContractRead } from 'wagmi'
+
+import { Icon } from 'src/components/Icon'
+import SWR_KEYS from 'src/constants/swrKeys'
+import { auctionAbi } from 'src/data/contract/abis'
+import { ProposalsResponse, getProposals } from 'src/data/graphql/requests/proposalsQuery'
+import { NounsProposalStatus } from 'src/data/graphql/sdk.generated'
+import { useDaoStore } from 'src/modules/dao'
+
+export const AuctionPaused = () => {
+  const { query, isReady } = useRouter()
+  const LIMIT = 20
+
+  const { auction } = useDaoStore((x) => x.addresses)
+  const auctionContract = useContract({ abi: auctionAbi, address: auction })
+  const { data: paused } = useContractRead({
+    abi: auctionAbi,
+    address: auction,
+    functionName: 'paused',
+  })
+
+  const { data } = useSWR<ProposalsResponse>(
+    paused && isReady ? [SWR_KEYS.PROPOSALS, query.token, query.page] : null,
+    (_, token, page) => getProposals([token], LIMIT, Number(page))
```

Comment:
there is a `proposalsWithCalldata` query with a smaller surface area of attributes that are returned if you want to swap this out! but i see that we're actually trying to reference the page of proposals so that makes sense too!

Reply:
I think reusing the proposals SWR key / request will allow us to use the cached data that gets queried for the activity tab right? let me know if I'm off on this though

Reply:
yes yes it will reference the cached data and also trigger a re-fetch, this is fine!